### ISSUE TITLE
Sample Issue with iOS

### ISSUE THREAD

**Issue by c-droege** [okta/samples-js-react-native#108]
Hi,
I played around with the samples downloaded from the admin area directly after creating a client. 
After running `npm install`, `pod install` and `npm run ios` the compilation of the App wasn't possible due to a problem with the OktaSdkBridge 

https://github.com/okta/okta-react-native/blob/f9e16a7ff6cdab3cae4db5fff5dd4e2c102ee7fb/ios/OktaSdkBridge/OktaSdkBridge.swift#L568

The problem has been resolved in Version 2.2.0 so I updated my package.json locally and after that everything was up and running.

Solved:
https://github.com/okta/okta-react-native/blob/9192e80781eb28c86bd5abde1c06e7f3105f1f04/ios/OktaSdkBridge/OktaSdkBridge.swift#L568

It would be awesome if you fix this and check if Android is still working.

Cheers, 
Chris

**Comment by oleggnidets-okta** [comment]
Hi there,
Thanks for the issue. 

I believe you have cached `Pods` folder, try to remove it and `Podfile.lock` and re-install with `pod install --repo-update`.
Moreover, I see [tests succeeded](https://app.travis-ci.com/github/okta/samples-js-react-native/builds/241799795), they're working on `2.1.1`.

Let us know whether it works for you.

**Comment by c-droege** [comment]
Hey, 
sorry for the delay. I doubt that my podfiles were cached. The installation of pods was a complete fresh installation. I had a problem with react-native before. It seems that the don't like spaces in a file path, resulting in a missing dependency while building the project.
After I fixed this by moving the project, the problem above occurred.  I deleted the ```Podfile.lock``` and the Pods folder  after that because I had the same idea like you.

Unfortunately the only thing the worked quite fast was to update the dependency. 

I can try your way as well and give you feedback when I have done.
